src/views/preview/project-view.jsx

```diff
@@ -273,9 +273,28 @@ class Preview extends React.Component {
 
                     const helpers = ProjectInfo[projectData[0].projectVersion];
                     if (!helpers) return; // sb1 not handled
-                    newState.extensions = helpers.extensions(projectData[0]);
+                    newState.extensions = Array.from(helpers.extensions(projectData[0]));
                     newState.modInfo.scriptCount = helpers.scriptCount(projectData[0]);
                     newState.modInfo.spriteCount = helpers.spriteCount(projectData[0]);
+                    if (helpers.cloudData(projectData[0])) {
```

Comment:
`helpers.cloudData(projectData[0])` is now getting called twice (here and below inside the showCloudDataAlert) line, maybe we can save it up here in a const?